stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -32,7 +32,7 @@
 public class Stripe {
 
     @VisibleForTesting
-    TokenCreator tokenCreator = new TokenCreator() {
+    TokenCreator mTokenCreator = new TokenCreator() {
```

Comment:
Decided to finally fully convert this class because of all the changes I've made. It's  a package-private variable, so there's no harm in changing names.